CONTRIBUTING.md

```diff
@@ -59,15 +59,15 @@ Other commit information:
 
 * [How to format the message][commit-message-message]
 
-## How to build the docker image (tag 18.04)
+## How to build the docker image (tag 22.04)
 
 ```bash
-docker build --no-cache --build-arg version=18.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:18.04 .
+docker build --network host --no-cache --build-arg version=22.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:22.04 .
```

Comment:
> --build-arg version=22.04 --build-arg strip=true

It looks like the image was initially designed to take the ubuntu version as a `build argument`, as well as whether to "strips" it or not. I do not see these arguments used in the image any more, so they could be removed to simplify the build command. In any case, I would think in most cases, more than the Ubuntu version will need be updated, for the image to work with a new Ubuntu, so it's ok not having this configuration.

Reply:
Latest version looks good!